chord_metadata_service/restapi/validators.py

```diff
@@ -31,3 +34,7 @@ def deconstruct(self):
 ontologyListValidator = JsonSchemaValidator(ONTOLOGY_CLASS_LIST)
 keyValueValidator = JsonSchemaValidator(KEY_VALUE_OBJECT)
 ageOrAgeRangeValidator = JsonSchemaValidator(AGE_OR_AGE_RANGE)
+quantity_validator = JsonSchemaValidator(schema=QUANTITY, format_checker=['uri'])
+tumor_marker_test_validator = JsonSchemaValidator(schema=TUMOR_MARKER_TEST)
+complex_ontology_validator = JsonSchemaValidator(schema=COMPLEX_ONTOLOGY, format_checker=['uri'])
+time_or_period_validator = JsonSchemaValidator(schema=TIME_OR_PERIOD, format_checker=['date-time'])
```

Comment:
Nitpick but variable names should be consistent, either all `camelCase` or all `snake_case`

Reply:
@romgrk yeah I know, it should be all `snake_case` for variables, can we do this in general code refactoring after mcode merged because it uses shared validators that are used in experiments, I don't want to introduce changes to experiments models on this branch